discover.js

```diff
@@ -0,0 +1,17 @@
+
+const WSConnection = require('./connections/ws')
+const SerialConnection = require('./connections/serial')
+
+async function listDevices({ ignoreSerial, ignoreWebsocket } = {}) {
```

Comment:
Would you consider making this a static method in the `LoupedeckDevice` class instead of a separate module? That would allow a more pleasant API in the form of:
```
import { LoupedeckDevice } from 'loupedeck'
const devices = await LoupedeckDevice.list()
```

Reply:
Implemented in [221d7ef](https://github.com/foxxyz/loupedeck/pull/12/commits/221d7ef9c8f3a716abceb59c300dbe3f160d8c5b)